drf_braces/fields/custom.py

```diff
@@ -65,3 +66,35 @@ class NumericField(ValueAsTextFieldMixin, fields.IntegerField):
 
 __all__ = [name for name, value in locals().items()
            if inspect.isclass(value) and issubclass(value, fields.Field)]
+
+
+class RoundedDecimalField(fields.DecimalField):
+    """
+    Currency field subclass of Decimal used for rounding currencies
+    to two decimal places.
+    """
+
+    def __init__(self, max_digits=None, decimal_places=2, *args, **kwargs):
+        super(fields.DecimalField, self).__init__(
+            max_digits=max_digits,
+            decimal_places=decimal_places,
+            *args, **kwargs
+        )
+
+    def to_internal_value(self, data):
+        return self.quantize(super(RoundedDecimalField, self).to_internal_value(data))
+
+    def quantize(self, data):
+        if self.decimal_places is None:
+            return data
+
+        context = getcontext().copy()
```

Comment:
this seems straight copy from DRF, even older versions - https://github.com/encode/django-rest-framework/blob/3.2.5/rest_framework/fields.py#L986

can we call `super` instead?